samples/client/petstore/typescript-fetch/builds/with-interfaces/runtime.ts

```diff
@@ -209,17 +209,9 @@ export function querystring(params: HTTPQuery, prefix: string = ''): string {
             }
             return `${encodeURIComponent(fullKey)}=${encodeURIComponent(String(value))}`;
         })
-        .filter(part => part.length > 0)
         .join('&');
 }
 
-export function mapValues(data: any, fn: (item: any) => any) {
```

Comment:
this should not be removed. did you generate the samples with the most recent master merged into your branch?

Reply:
I'm completely stumped now. I tried merging upstream/master into my branch before your review. After that, I pulled upstream master into my master and merged it into my branch. This got me back to where I was before @wing328 reviewed my changes.

Reply:
@Teyras are you sure you re-generated the samples *after* merging `origin/master`?